### PR TITLE
[full-ci] Postprocessing Bulk restart

### PR DESCRIPTION
**Pull request by kobergj**
Allows restarting all uploads currently in a specific postprocessing step

### CODE REVIEW

File: services/postprocessing/pkg/command/postprocessing.go [hunk 1]
```diff
@@ -20,10 +19,15 @@ func RestartPostprocessing(cfg *config.Config) *cli.Command {
...
-				Usage:    "the uploadid to restart",
+				Name:    "upload-id",
+				Aliases: []string{"u"},
+				Usage:   "the uploadid to restart. Ignored if unset.",
+			},
+			&cli.StringFlag{
+				Name:    "step",
+				Aliases: []string{"s"},
+				Usage:   "restarts all uploads currently in that specific postprocessing step. Ignored if upload-id is set.",
+				Value:   "finished", // Is this default sane? Calling `ocis postprocessing restart` without any arguments will restart all finished uploads.
```
Comment (butonic) [inline services/postprocessing/pkg/command/postprocessing.go:30]:
```suggestion
				Usage:   "restarts all uploads in the given postprocessing step. Ignored if upload-id is set.",
				Value:   "finished", // Calling `ocis postprocessing restart` without any arguments will restart all uploads that are finished but failed to move the uploed from the upload area to the blobstore.
```

**Review by butonic — CHANGES_REQUESTED** [review]
Please update the README.md regarding how to restart all uploads in a given step.

Also, there is no command to get a list of the state every upload is in, right? We could look into external stores, but the default memory does not work, right? Hm, for in memory all ongoing uploads are lost ... and there is no way cli to pick up the upload sessions from disk again. they can only be cleaned. Shoud we change the default to nats-js-kv?

Ah ok, by giving the upload-id the postprocessing can always be restarted ...

**Review by butonic — APPROVED** [review]

**Review by micbar — APPROVED** [review]

**Comment by saw-jan** [comment]
Need to adjust MOVE related API test expectations again. I could have pushed the changes but the PR is from the fork

**Comment by kobergj** [comment]
I added you to the fork. Can you accept the invite and retry?

**Comment by saw-jan** [comment]
> Need to adjust MOVE related API test expectations again. I could have pushed the changes but the PR is from the fork

Thanks, I will push the test changes

**Comment by saw-jan** [comment]
Updated the tests. :+1: from the CI. :sparkles:

**Pull request merged**